simpleclient_hibernate/src/main/java/io/prometheus/client/hibernate/HibernateStatisticsCollector.java

```diff
@@ -0,0 +1,282 @@
+package io.prometheus.client.hibernate;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.CounterMetricFamily;
+import io.prometheus.client.GaugeMetricFamily;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.hibernate.SessionFactory;
+import org.hibernate.stat.Statistics;
+
+/**
+ * Collect metrics from Hibernate statistics.
+ * <p>
+ * Usage example:
+ * <pre>
+ * new HibernateStatisticsCollector(sessionFactory).register();
```

Comment:
Yes. Especially in application servers. Actually I was thinking about adding a label identifying the `SessionFactory` to the metrics. But it looks like there isn't something like a "name" of a `SessionFactory`.